node/cn/filters/api.go

```diff
@@ -243,13 +244,17 @@ func RPCMarshalHeader(head *types.Header, isEnabledEthTxTypeFork bool) map[strin
 		"hash":             head.Hash(),
 	}
 
-	if isEnabledEthTxTypeFork {
+	if rules.IsEthTxType {
 		if head.BaseFee == nil {
 			result["baseFeePerGas"] = (*hexutil.Big)(new(big.Int).SetUint64(params.ZeroBaseFee))
 		} else {
 			result["baseFeePerGas"] = (*hexutil.Big)(head.BaseFee)
 		}
 	}
+	if rules.IsRandao {
+		result["reandomReveal"] = hexutil.Bytes(head.RandomReveal)
```

Comment:
I meant this. Sorry for the wrong pointer.
```suggestion
		result["randomReveal"] = hexutil.Bytes(head.RandomReveal)
```

Reply:
Fixed. Thank you!